lib/web/userRouter.js

```diff
@@ -90,18 +90,20 @@ UserRouter.get('/me/export', function (req, res) {
           ownerId: user.id
         }
       }).then(function (notes) {
-        let list = []
+        let filenames = {}
         async.each(notes, function (note, callback) {
-          let title
-          let extension = ''
+          let basename = note.title.replace(/\//g, '-') // Prevent subdirectories
+          let filename
+          let suffix = ''
           do {
-            title = note.title + extension
-            extension++
-          } while (list.indexOf(title) !== -1)
+            let seperator = typeof suffix === 'number' ? '-' : ''
+            filename = basename + seperator + suffix + '.md'
+            suffix++
+          } while (filenames[filename] !== undefined)
```

Comment:
```suggestion
          } while (filenames[filename])
```

Reply:
(this is really nitpicking, if it stays the same and we merge it, it's really fine as well)

Reply:
Updated the code. :)